.github/workflows/unit-integration-test.yml

```diff
@@ -74,6 +74,8 @@ jobs:
           mvn -ntp -U clean
 
       - name: Run unit and integration tests
+        env:
+          MAVEN_OPTS: "-Xmx1G -Xms1G"
```

Comment:
This is entirely unrelated to the change, but I did receive at least one OOM as part of the regular CI checks. I'd be happy to pull this out of the changeset if this or something similar will land elsewhere.